### PR TITLE
TF Lite profiling module warning fix

### PR DESCRIPTION
**Pull request by Dayananda-V**
profiling module warning fix

### CODE REVIEW

File: tensorflow/lite/profiling/profile_summarizer_test.cc [hunk 1]
```diff
@@ -86,15 +84,6 @@ class SimpleOpModel : public SingleOpModel {
   int output_;
 };
 
-void SimpleOpModel::Init(
```
Comment (jdduke) [inline tensorflow/lite/profiling/profile_summarizer_test.cc]: I don't fully understand this removal? Why?

**Review by jdduke — CHANGES_REQUESTED** [review]

File: tensorflow/lite/profiling/profile_summarizer_test.cc (hunk 1 above)
Comment (Dayananda-V) [reply tensorflow/lite/profiling/profile_summarizer_test.cc]: unused code cleanup.

Comment (jdduke) [reply tensorflow/lite/profiling/profile_summarizer_test.cc]: `m.Init(RegisterSimpleOp);` is called below, in ProfileSummarizerTest.Interpreter.

Comment (Dayananda-V) [reply tensorflow/lite/profiling/profile_summarizer_test.cc]: m.Init(RegisterSimpleOp); this is L76 method invocation, after remove this code also test case run is success.

Comment (jdduke) [reply tensorflow/lite/profiling/profile_summarizer_test.cc]: That's because that test is behind a build define.

Comment (Dayananda-V) [reply tensorflow/lite/profiling/profile_summarizer_test.cc]: so how to deal this now?

Comment (jdduke) [reply tensorflow/lite/profiling/profile_summarizer_test.cc]: I've removed the build define usage, so we can close this PR.

**Comment by jdduke** [comment]
Closing as this code is no longer hidden by a build flag.

**Pull request closed without merging**